frontend/packages/console-app/console-extensions.json

```diff
@@ -1090,6 +1308,55 @@
       }
     }
   },
+  {
+    "type": "console.navigation/resource-ns",
+    "properties": {
+      "perspective": "admin",
+      "section": "storage",
+      "id": "volumegroupsnapshots",
+      "insertAfter": "volumesnapshotcontents",
+      "name": "%console-app~VolumeGroupSnapshots%",
+      "model": {
+        "group": "groupsnapshot.storage.k8s.io",
+        "version": "v1beta1",
+        "kind": "VolumeGroupSnapshot"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/resource-cluster",
+    "properties": {
+      "perspective": "admin",
+      "section": "storage",
+      "id": "volumegroupsnapshotclasses",
+      "insertAfter": "volumegroupsnapshots",
+      "name": "%console-app~VolumeGroupSnapshotClasses%",
+      "model": {
+        "group": "groupsnapshot.storage.k8s.io",
+        "version": "v1beta1",
+        "kind": "VolumeGroupSnapshotClass"
+      }
+    }
+  },
+  {
+    "type": "console.navigation/resource-cluster",
+    "properties": {
+      "id": "volumegroupsnapshotcontents",
+      "section": "storage",
+      "name": "%console-app~VolumeGroupSnapshotContents%",
+      "insertAfter": "volumegroupsnapshotclasses",
+      "model": {
+        "group": "groupsnapshot.storage.k8s.io",
+        "kind": "VolumeGroupSnapshotContent",
+        "version": "v1beta1"
+      }
+    },
+    "flags": {
+      "required": [
+        "CAN_LIST_VGSC"
+      ]
+    }
```

Comment:
we don't need flags for other nav-items (VolumeGroupSnapshots & VolumeGroupSnapshotClasses) ??

Reply:
"A VolumeSnapshotContent is a snapshot taken from a volume in the cluster that has been provisioned by an administrator. It is a resource in the cluster just like a PersistentVolume is a cluster resource."

https://kubernetes.io/docs/concepts/storage/volume-snapshots/

This is actually the representation of Snapshot content in the backend containing the data. Hence it needs to be protected. Other resources are more of a logical representation that don't need to be protected. That is how i understand it